pkg/cmd/server/api/validation/validation.go

```diff
@@ -105,11 +105,11 @@ func ValidateCertInfo(certInfo api.CertInfo, required bool, fldPath *field.Path)
 	return allErrs
 }
 
-func ValidateServingInfo(info api.ServingInfo, fldPath *field.Path) ValidationResults {
+func ValidateServingInfo(info api.ServingInfo, certificatesRequired bool, fldPath *field.Path) ValidationResults {
```

Comment:
I really prefer the zero value to be the "safe" thing to do (passing in `false` should do the correct thing):

`certificatesRequired` vs `skipCertificateVerification`

Reply:
I prefer positive arguments, negatives are more prone to confusion, which is our dominant style.

Reply:
maybe at least rename to: enforceCertificateVerification ?